### DISCUSSION TITLE
Open TV urgently needs your support!

### DISCUSSION THREAD

**Discussion by Fredolx** [Fredolx/open-tv#69] [Announcements]
Hi, my name is Frédéric Lachapelle, developer of Open TV.

I've been developing Open TV alone and without much support for 3 years because I believed the world needed a bloat-free, simple, ultra-fast and most importantly open-source IPTV app.

I'm glad to have successfully launched 1.0, and I want to deliver more updates and features for everyone, but for that, **I need your help.** At the moment, Open TV is not sustainable. Without your support, I may not be able to continue the development of Open TV at the current pace or at all.

### I am currently looking for a permanent job, please refer me to your connections/employer if you are in Canada. [Click here for my LinkedIn](https://www.linkedin.com/in/frederic-lachapelle)

**Comment by Schrotty74** [comment]
Please add hide channels or groups.

**Reply by bkayy101-cmyk** [reply]
Is there possibility to add Stalker codes

**Reply by G4uol** [reply]
Great program - paid via Store - look forward to future updates

**Comment by spikerguy** [comment]
Donated a small amount.

Cannot appreciate your work on this app much. 

Keep up the good work.

**Reply by Fredolx** [reply]
Thank you!

**Comment by llKRAMERll** [comment]
Great app! Purchased on MS Store. Recommending to friends and others, keep up the good work. Plans for iOS/Android?

**Reply by Fredolx** [reply]
It's in development and a beta is ongoing for Android. I'm waiting on Google for a full release

**Reply by tusx** [reply]
> It's in development and a beta is ongoing for Android. I'm waiting on Google for a full release

might there be any chances of it being available on F-droid? thanks

**Comment by darylraj** [comment]
great app, enjoy using it on my Pc when working, ahah, great for xtream codes and M3u, UI could include more user friendly epg setup. as mentioned, great app

**Comment by snowqueen92** [comment]
Dude, your TV app ROCKS! 

I wonder if there's a way to integrate it into Jellyfin, that'd be awesome. Yours is much more stable than theirs.

**Comment by Fredolx** [comment]
### I am currently looking for a permanent job, please refer me to your connections/employer if you are in Canada. [Click here for my LinkedIn](https://www.linkedin.com/in/frederic-lachapelle)

**Comment by EdmundDuckDuck** [comment]
Hi it work for my playlist on Linux, happy with it.
I don't see anything where to buy it on windows, not that I want that but to let you know.
Can I get a .deb installer somewhere, sometimes linux progs suddenly disappear so I prefer a program that I install myself.

**Comment by magmasystems** [comment]
Hi Fred, Great work on version 1.8. I can now see the series. I would like to donate, but I don't want to use any of the methods that you currently use. Can you accept Venmo? If so, please share the details. You can send me a private message here if you want, or just include your Venmo details.

**Reply by Fredolx** [reply]
We don't have venmo/cashapp in Canada. If you have any other ways to donate, please let me know. I will add it if it's feasible. Thank you!

**Reply by magmasystems** [reply]
I donated via PayPal this morning. Thanks for all of your work!

**Comment by EdmundDuckDuck** [comment]
There is a difference between the m3u lists and xtream, in the m3u the movies/vod/series are together while in xtream the series are separated. No big deal, my list works so I will make a donation.

**Reply by Fredolx** [reply]
it's a limitation of M3U, not the app. There is no way to know if something is a series on M3U

**Comment by d1videbyzero** [comment]
Which donation method do you prefer?

**Reply by Fredolx** [reply]
paypal and github

**Comment by AnonNDDL** [comment]
Cben Genial ca! je suis a Notre-dame-du-la dans le Témis on pas de reception OTA les 2 antennes son mortes depuis un ans.
Ton soft me permet dfe restreamer a mes voisin qui on pas le cable Les Nouvelle a temps plein!

**Reply by Fredolx** [reply]
Merci :)

**Comment by AlphaSeekr** [comment]
Great app. Donated through Paypal. Please implement Picture in picture and EPG upload.

**Comment by TokerAnon** [comment]
Donated last night - superb software 👌

**Reply by Fredolx** [reply]
Thank you :)

**Comment by RainGater** [comment]
I bought it in the Microsoft store to support the development although it's a very small amount.  

Great app, btw!  Works very well and not much buffering and appreciate your time/effort into the development of this app.

One suggestion:  If you can provide a way to back up all the settings (including sources, just like TiviMate does) and NOT just favorites, that will be very useful.

One minor bug: When I toggle Movies/Vods and Series OFF, it comes back active when I launch the app again.  The reason being I don't really care about Movies and Series and every time I have to deselect those when I search. It's a minor annoyance and if you can retain the user selections, that will be helpful.

**Reply by Fredolx** [reply]
I could persist them, it wouldn't be a bad idea

**Comment by RainGater** [comment]
Fred TV on Android doesn't work.  I loaded the same Xtream URL and credentials that I use on Windows but nothing loads in the Fred TV app.  No error as well.  Please check it out.

FWIW, it works great on Windows. 

https://play.google.com/store/apps/details?id=dev.fredol.open_tv

**Reply by Fredolx** [reply]
I'll look into it, thank you :)

**Comment by 100torri-ita** [comment]
Please EPG support. Thx for your effort!

**Reply by Fredolx** [reply]
It's already here but a bit basic, only for Xtream and you need to right click on individual channels

**Comment by lucasfffgit** [comment]
I did pay for the license on windows store as trial expired, hope most of it goes to you. I will contribute more in the future as the app is great. I would like to only see more EPG integration, like in favourites tab where I have all my channels maybe below the name of the channel there could be EPG of whats playing now. Thanks

**Reply by Fredolx** [reply]
Thank you :)

**Comment by lucasfffgit** [comment]
Hi Why is the multiple channels not working anymore? It was working on trial version, now i paid and suddenly when i click other channel then the running one switches off? It was one of the best features to run 2 channels at once.

**Reply by Fredolx** [reply]
You can change the max streams value in settings

<img width="670" height="369" alt="image" src="https://github.com/user-attachments/assets/71df8d19-be4d-467d-92bc-07d2b220ec52" />

Apparently most users did not use the feature and found it annoying, but maybe it was a loud minority, so it's "off" by default unless you increase the amount of max_streams

**Reply by lucasfffgit** [reply]
amazing thanks, i love this setting to be able to watch 2 channels at once, didnt see any other iptv app offer that, thats why i chose yours Fred!

**Reply by lucasfffgit** [reply]
although now when i run the second channel, the first one stops working or stutters a lot, maybe its just temporary. before the 2 streams were running very good together

**Comment by alpha42net** [comment]
Salut, c'est moi ou le multiscreen ne fonctionne plus, c'était la fonction que j'aime le plus

**Reply by Fredolx** [reply]
Tu dois augmenter le nombre de max_streams sur tes sources. Beaucoup d'users de plaignaient que le multi stream c'était bizarre alors j'ai mis une limite de 1 stream par source par défaut.

**Reply by Fredolx** [reply]
<img width="670" height="369" alt="image" src="https://github.com/user-attachments/assets/b36c3010-3e7b-4967-80b2-4d5a603139ac" />

c'est la valeur max streams que tu vois dans le screenshot

**Reply by alpha42net** [reply]
excellent, merci et en passent, je viens de te faire un don, tu le mérite, excellente application pour le iptv. à quand une version samsung avec le multiscreen ;P pour un fan de sports comme moi c'Est génial

**Reply by Fredolx** [reply]
Merci :)

**Comment by EdmundDuckDuck** [comment]
Hi Fredolx, EPG search and program to download option would be amazing.
But I am already happy with what it is now, Thank you for that.

**Reply by Fredolx** [reply]
You can already download EPG vods on Xtream sources, if the option is not availaible for you, it's a bug

**Reply by EdmundDuckDuck** [reply]
I have seen EPG on the live streams but what I mean is an option to search "system wide" trough all live streams in one search.
(just as in satellite receivers).
BTW the latest 19.1 version does not work on my Linux-laptop but it works fine on my linux-desktop.
On the laptop the video's stutter between one or two frames and the video does not play, sometimes the sound plays.
If I must provide you with some more info tell me what to do.
Regards

**Comment by EdmundDuckDuck** [comment]
Strange ( for me ) download happening.
Downloading/recording seems to work well in the latest version.
however I downloaded a 4k movie that didn't play well, so I thought I download it and then I can play it without problems.
Strangely enough the stuttering also happened in the downloaded file.

**Comment by pinarteks-rgb** [comment]
Önceleri Görüntü normaldi Ama şimdi Görüntü önünde perde var sebebi nedir

**Comment by jairmoreno** [comment]
Hey! I just bought it in the Microsoft Store to support you. Hope it helps!

BTW, could you enable the function to add a category to Favorites? It would really improve the experience on the app.

Thanks!

**Reply by Fredolx** [reply]
Thank you :)

**Comment by Jisifus** [comment]
Glad to help, awesome job on 1.9.1. Keep up the great work.

**Comment by 0-bartos** [comment]
Fred
I just donated to your cause and would also like to use it on a Windows machine. I know it's not much, but can I get a functional copy for my Windows machine?

**Reply by Fredolx** [reply]
Yes, https://github.com/Fredolx/open-tv/releases/download/v1.9.1/Fred.TV_1.9.1_x64_en-US.msi

**Comment by 0-bartos** [comment]
Thank you for the link and your welcome

Bart....
________________________________

**Reply by Fredolx** [reply]
Thank you again for your donation :)

**Comment by pinarteks-rgb** [comment]
Videolarda neden Filigran var ?

**Reply by Fredolx** [reply]
It's not because of Fred TV, probably your provider

**Comment by traidor** [comment]
Hi. I can contribute with Romanian translation if possible.
